tests_multiselect.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
```

Comment:
Here we just check out the Jasmine tool. Nothing productive here at the moment, just a copy.